### DISCUSSION TITLE
Newbe on Doodba Deploys

### DISCUSSION THREAD

**Discussion by YnievesDotNet** [Tecnativa/doodba#507] [General]
Thanks for this environment, I'm are studying to use that, but I need know some particular requirements.

1. Need use some GHub repositories of addons, some of OCA and two or three personal, how I can put then in the directories of odoo.

2. Another thing is about the config, in the youtube video PBaeza tell about some particular .conf files to make a one big, that I need probe calling the addons added in the first question.

3. As last point for now, I make a one deploy for some DB, 5 or 6, and some addons aren't use for all db, is possible if I update a custom addons their are updated only in the db was use that, not all DB?

Thans everybody and good project are Doodba.

**Comment by pedrobaeza** [comment]
Can you please open one discussion for each point? You can let this one for the first point

**Reply by YnievesDotNet** [reply]
Ok, thanks, and good project @pedrobaeza, I see your video and I wan use the Doodba on my environment.

**Comment by sgheller** [comment]
> 1. Need use some GHub repositories of addons, some of OCA and two or three personal, how I can put then in the directories of odoo.

[Doodba docs](https://github.com/Tecnativa/doodba#optodoocustomsrc)

**Reply by YnievesDotNet** [reply]
Thanks, I read now
